src/main/java/uk/gov/hmcts/dm/domain/StoredDocument.java

```diff
@@ -51,6 +53,10 @@
     @Setter
     private boolean deleted;
 
+    @Getter
+    @Setter
+    private boolean hardDeleted;
```

Comment:
question what happends if somthing is "hardDeleted" but not "deleted" in theroy is possible to still receive the document(without the binary)